fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,142 @@
+# import datetime
+import json
+
+# import yaml
+import time
+from typing import Any
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from datadog_checks.base.utils.db import QueryManager
+# from requests.exceptions import ConnectionError, HTTPError, InvalidURL, Timeout
+# from json import JSONDecodeError
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
+metrics_list = ['accuracy']
+tags = list()
+
+# Connection parameters for Fiddler. Currently hard coded to get some testing done. Will be parameterizing this later.
+URL = 'https://demo.trial.fiddler.ai'
+ORG_ID = 'demo'
+AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
+
+client = fdl.FiddlerApi(url=URL, org_id=ORG_ID, auth_token=AUTH_TOKEN)
+
+
+def create_tags(**tags_in):
+    tags.clear()
+    tags.append("project:" + tags_in['project'])
+    tags.append("model:" + tags_in['model'])
+    if len(tags_in) > 2:
+        tags.append("feature:" + tags_in['feature'])
+
+
+class FiddlerCheck(AgentCheck):
+
+    # This will be the prefix of every metric and service check the integration sends
+    __NAMESPACE__ = 'fiddler'
+
+    def __init__(self, name, init_config, instances):
+        super(FiddlerCheck, self).__init__(name, init_config, instances)
+
+        # Use self.instance to read the check configuration
+        # self.url = self.instance.get("url")
```

Comment:
```suggestion
```

Reply:
@yzhan289 - Andrew, the can_connect service check is already there in the service_checks.json file. 

[
    {
        "agent_version": "6.0.0",
        "integration":"fiddler",
        "groups": ["host", "instance"],
        "check": "fiddler.can_connect",
        "statuses": ["ok", "critical"],
        "name": "Can Connect",
        "description": "Returns `CRITICAL` if the Agent is unable to connect to and collect metrics from the monitored Fiddler instance. Returns `OK` otherwise."
    }
]